pupil-spa/src/app/services/question/question.service.ts

```diff
@@ -12,17 +12,28 @@ const configKey = 'config';
 export class QuestionService {
 
   protected currentQuestion;
-  protected questions;
-  protected config: Config;
+  protected config;
+
+  protected questions = [
+    {'order': 1, 'factor1': 2, 'factor2': 5},
+    {'order': 2, 'factor1': 11, 'factor2': 2},
+    {'order': 3, 'factor1': 5, 'factor2': 10},
+    {'order': 4, 'factor1': 4, 'factor2': 4},
+    {'order': 5, 'factor1': 3, 'factor2': 9}
+  ];
 
   constructor(protected storageService: StorageService,
               protected speechService: SpeechService) {
     this.currentQuestion = 0;
 
-    // Re-read the stored questions on page refresh
-    if (this.storageService.getItem(questionKey) && this.storageService.getItem(configKey)) {
-      this.initialise();
-    }
+    const config = new Config();
+    config.loadingTime = 2;
```

Comment:
Shouldn't this be 3?  I may be mistaken though...

Reply:
Yup, you’re right! A normal config for pupils has `loadingTime = 3`; corrected in a new commit 👍